content/en/docs/concepts/scheduling-eviction/pod-scheduling-readiness.md

```diff
@@ -89,6 +89,24 @@ The metric `scheduler_pending_pods` comes with a new label `"gated"` to distingu
 has been tried scheduling but claimed as unschedulable, or explicitly marked as not ready for
 scheduling. You can use `scheduler_pending_pods{queue="gated"}` to check the metric result.
 
+## Mutable Pod Scheduling Directives
+
+{{< feature-state for_k8s_version="v1.27" state="beta" >}}
+
+Pods with scheduling gates have mutable scheduling directives, with certain constraints. 
```

Comment:
You can mutate scheduling directives of Pods while they have scheduling gates, with certain constraints.